src/pivotal-ui/components/forms/package.json

```diff
@@ -5,5 +5,5 @@
     "@npmcorp/pui-css-bootstrap": "^2.0.0",
     "npm-user-validate": "^0.1.2"
   },
-  "version": "3.2.0"
+  "version": "3.2.3"
```

Comment:
I think I would've made it one, but the only place it changes is a component we haven't used yet.